airbyte-integrations/connectors/source-chargify/source_chargify/spec.json

```
@@ -0,0 +1,21 @@
+{
+  "documentationUrl": "https://docsurl.com",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Chargify Spec",
+    "type": "object",
+    "required": ["api_key", "domain"],
+    "additionalProperties": false,
+    "properties": {
+      "api_key": {
+        "type": "string",
+        "description": "Chargify API Key.",
+        "airbyte_secret": true
+      },
+      "domain": {
```

```suggestion
      "server": {
```
The API doc uses the word `server` not domain 😄